fastestimator/pipeline.py

```diff
@@ -482,6 +488,31 @@ def __enter__(self) -> Union[DataLoader, tf.data.Dataset]:
             collate_fn = self.collate_fn
             if collate_fn is None and self.pad_value is not None:
                 collate_fn = self._pad_batch_collate
+            # Default ExapandDataset Sampler
+            if self.ctx_train_steps_per_epoch is not None and self.ctx_mode == "train":
+
+                if self.ctx_shuffle == True:
```

Comment:
instead of this if branch, just set shuffle=self.ctx_shuffle when constructing the sampler